tests/cxx/relative_exported_mapped_include.cc

```diff
@@ -7,6 +7,9 @@
 //
 //===----------------------------------------------------------------------===//
 
+// IWYU_ARGS: -Xiwyu --mapping_file=tests/cxx/relative_exported_mapped_include.imp\
```

Comment:
Hmm, this line is actually too long. Unfortunately, I think you need to turn it around into:
```
// IWYU_ARGS: -I tests/cxx/subdir -Xiwyu \
//            --mapping_file=...
```
It looks a little atypical, but it works better. We're pretty strict with the 80-column limit.